core/src/main/scala/io/gearpump/util/Constants.scala

```diff
@@ -119,9 +120,11 @@ object Constants {
   //Service related
   val GEARPUMP_SERVICE_HTTP = "gearpump.services.http"
   val GEARPUMP_SERVICE_HOST = "gearpump.services.host"
+  val GEARPUMP_SERVICE_SUPERVISOR_PATH = "gearpump.services.supervisor-actor-path"
 
   //The folders are under ${GEARPUMP_HOME}
-  val EXECUTOR_CLASSPATH_WHILTELIST = Array("conf", "lib")
+
+  val daemonPath = Array("lib" + File.separator + "daemon")
```

Comment:
service classpath is not included in Executor. 
Executor will inherit  classpath from worker
```
  val daemonClassPath = Seq(
    "${PROG_HOME}/conf",
    "${PROG_HOME}/lib/daemon/*"
  )
```